pkg/cloud/services/securitygroup/securitygroups_test.go

```diff
@@ -90,6 +92,29 @@ func TestReconcileSecurityGroups(t *testing.T) {
 				},
 			},
 			expect: func(m *mocks.MockEC2APIMockRecorder) {
```

Comment:
We should also have a test for reconciliation if the rules are already gone – in that case, no requests apart from `DescribeSecurityGroups` and listing its rules should be done.

Reply:
Added a test case to cover that.